src/worldfactory.cpp

```diff
@@ -1510,4 +1515,4 @@ void worldfactory::delete_world( const std::string &worldname, const bool delete
     } else {
         get_world( worldname )->world_saves.clear();
     }
-}
+}
```

Comment:
Add a blank line at the end of this file.

Reply:
Thank you, I was a bit sleep deprived yesterday :)